app/models/manageiq/providers/redhat/infra_manager/vm/operations/power.rb

```diff
@@ -4,7 +4,11 @@ def validate_pause
   end
 
   def raw_start
-    with_provider_object(&:start)
+    start_with_cloud_init = custom_attributes.find_by(:name => "miq_provision_boot_with_cloud_init")
+    with_provider_object do |rhevm_vm|
+      rhevm_vm.start { |action| action.use_cloud_init(true) if start_with_cloud_init }
+    end
```

Comment:
Does it makes sense to move this into a method on the VmRedhat so that we don't need to have native object handling here?